src/main.rs

```diff
@@ -67,7 +75,18 @@ fn validate_simulation() {
 
 /// Simulate the rabbit population
 fn simulate(starting_rabbits: i128) -> i32 {
-    todo!()
+    let mut count = 0;
+    let mut current_popn = starting_rabbits;
+    while current_popn != 1{
+        if current_popn % 2 == 0{
+            current_popn = current_popn / 2;
+            count += 1;
+        }else{
```

Comment:
Also, remember to run `cargo fmt` since it will clean up your code :)

Reply:
I ran cargo fat and nothing happened. How does it clean up my code?

Reply:
`cargo fmt` as in format :)